tests/core/test_core_device_containers.cpp

```diff
@@ -179,45 +179,71 @@ TEST_F(core_device_container_test, resizable_vector_buffer) {
     EXPECT_EQ(device_vector.back(), 15);
 
     device_vector.assign(20, 123);
-    EXPECT_EQ(device_vector.size(), static_cast<vector_size_type>(20));
```

Comment:
The size check should still remain, no? :confused:

Reply:
Yes, not sure why this happened.